src/v2/Apps/Artist/Routes/Overview/Components/ArtistArtworkFilter.tsx

```diff
@@ -24,18 +25,31 @@ const ArtistArtworkFilter: React.FC<ArtistArtworkFilterProps> = props => {
   // we still want to render the rest of the page.
   if (!hasFilter) return null
 
+  const sortOptions = [
+    { value: "-has_price,-prices", text: "Price (desc.)" },
+    { value: "-has_price,prices", text: "Price (asc.)" },
+    { value: "-partner_updated_at", text: "Recently updated" },
+    { value: "-published_at", text: "Recently added" },
+    { value: "-year", text: "Artwork year (desc.)" },
+    { value: "year", text: "Artwork year (asc.)" },
+  ]
+
+  const defaultSortValue =
+    getENV("DECAYED_MERCH_V3") === "experiment"
+      ? "-decayed_merch_v2"
```

Comment:
This is the kinda confusing part due to the mismatch in test and param naming now.

Reply:
yea! eh, i'll just leave it, will only be a few days